Std/Data/BitVec/Bitblast.lean

```diff
@@ -0,0 +1,180 @@
+/-
+Copyright (c) 2023 by the authors listed in the file AUTHORS and their
+institutional affiliations. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Harun Khan, Abdalrhman M Mohamed, Joe Hendrix
+-/
+import Std.Data.BitVec.Folds
+
+/-!
+# Bitblasting of bitvectors
```

Comment:
Could we have either at least one sentence, or a reference, for what exactly "bitblasting" means?

I know mathematicians are not meant to care about this file, but they may be curious. :-)